Source/SAAudioQueue.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  SAAudioQueue.swift
+//  SwiftAudioPlayer
+//
+//  Created by Joe Williams on 09/03/2021.
+//
+
+import Foundation
+
+struct SAAudioQueue<T> {
```

Comment:
Could rename to simply `SAQueue<T>`

Reply:
Yeah I think SAQueue could be simpler :)